django_select2/static/django_select2/django_select2.js

```diff
@@ -1,4 +1,15 @@
-(function ($) {
+/* global define, jQuery */
+(function (factory) {
```

Comment:
I am not so firm with modern day JavaScript. Why do we need all this boilerplate?

Reply:
It's one for amd, one for commonjs and one for browser globals. It's known as UMD

Reply:
You gotta give me more. Consider me a JS noob. Is that really needed or just a nice to have?

I really want to keep this simple... I guess you know why ;)

Reply:
it's so it works with `webpack`, `require.js` and as a django static asset

Reply:
Can you add a comment where the wrapper snippet is coming from. That way future contributors will know whats up.